src/Components/Shifting/ShiftingBoard.tsx

```diff
@@ -204,60 +203,29 @@ const ShiftCard = ({ shift, filter }: any) => {
           </div>
           {filter === "TRANSFER IN PROGRESS" && shift.assigned_facility && (
             <div className="mt-2">
-              <Button
+              <ButtonV2
                 size="small"
-                variant="outlined"
-                fullWidth
+                variant="secondary"
+                border={true}
```

Comment:
The button's width looks smaller than the View Details button which is right above it.

```suggestion
                border
                className="w-full"
```